backend/CuratedLists/views.py

```diff
@@ -133,10 +133,13 @@ def all_curations(request):
         response = HttpResponseNotAllowed(['GET'])
     else:
         try:
-            curations = Curation.objects.all()
+            curations = json.loads(serializers.serialize('json', Curation.objects.all())
+            for cur in curations:
```

Comment:
`curation` instead of `curations`? Semantic naming.

Reply:
I called it curations because it isn't a single curation. It is a list of all curations. Should I call it curationList instead?

Reply:
No, no I meant the loop variable.